AUTHORS

```diff
@@ -8,7 +8,7 @@ The project is continuation of DOSBox project, authored by:
 Sjoerd v.d. Berg <harekiet>
 Peter Veenstra <qbix79>
 Ulf Wohlers <finsterr>
-Tommy Fr�ssman <fanskapet>
+Tommy Frössman <fanskapet>
```

Comment:
Hmm, this looks weird, as if Unicode was actually replaced with some other encoding…

I'll add UTF-8 BOM and see if it clears the issue on your side.

Reply:
I don't think you can trust that display, it is a mix of codepage and utf-8

github "sees" the first codepage, and translates to UTF-8
the actual UTF-8 is translated from codepage ( which it is not) to UTF-8
this is why the replaced looks correct and the replacement incorrect
I suspect that recording the lines would result in first displaying correctly and second as junk

[What does the raw file look like? ](https://raw.githubusercontent.com/FeignInt/dosbox-staging/fi/ISO-8853-1_to_UTF-8/AUTHORS)

It looks like this started in 2003

0a38163d 2fc3c4b3

This is not pretty, but I hope demonstrates
```
for i in 0a38163d 2fc3c4b3;
do
  git grep . $i -- AUTHORS | file -
  git grep . $i -- AUTHORS | cat  -
  git grep . $i -- AUTHORS | cat  -A -
  git grep . $i -- AUTHORS | uconv -f ISO-8859-1 -t UTF-8
  git grep . $i -- AUTHORS | uconv -f ISO-8859-1 -t UTF-8 | cat -A -
done
```
it was ASCII text, then became ISO-8859

when viewing the file with a text editor, such as vim, it will correctly detect and convert to UTF-8
>"AUTHORS" [converted] 14L, 299C

The likes of grep, cat, less won't convert the codepage, the terminal then assumes the codepage is UTF-8 and as a result displays 'junk'


see: 
https://unix.stackexchange.com/questions/78776/characters-encodings-supported-by-more-cat-and-less

Reply:
Yup, seems to be a GitHub issue, depending on the context it will show broken characters in random locations.

e.g. in this PR correct glyphs are shown exactly the opposite
I re-created these files (they have same checksums as the ones in this PR), and when *creating* the PR or looking at commit they are shown correctly: cba7b3f34913aaf24958533d60351b9616159d36 - but in some contexts they are shown as properly-looking glyphs both on the left and right.